<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Massimo Callegari</author>
 <comment>
  Q Light Controller Plus
  audioeditor.ui

  Copyright (c) 2015 Massimo Callegari

  Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0.txt

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
 </comment>
 <class>AudioEditor</class>
 <widget class="QWidget" name="AudioEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="12" column="0" colspan="5">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Playback mode</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="m_singleCheck">
          <property name="text">
           <string>Single shot</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_loopCheck">
          <property name="text">
           <string>Loop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Channels</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QLabel" name="m_durationLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QLabel" name="m_srateLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="m_nameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Audio name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QToolButton" name="m_fileButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Audio device</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Fade out</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="m_filenameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QLineEdit" name="m_fadeInEdit">
       <property name="text">
        <string notr="true">.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLineEdit" name="m_nameEdit">
       <property name="toolTip">
        <string>Name of the function being edited</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" colspan="2">
      <widget class="QComboBox" name="m_audioDevCombo"/>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QLabel" name="m_channelsLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QLabel" name="m_bitrateLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bitrate</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QLineEdit" name="m_fadeOutEdit">
       <property name="text">
        <string notr="true">.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="m_previewButton">
       <property name="toolTip">
        <string>Play the audio file</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/player_play.png</normaloff>:/player_play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QToolButton" name="m_speedDialButton">
       <property name="toolTip">
        <string>Show/Hide speed dial window</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/speed.png</normaloff>:/speed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sample rate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="5">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fade in</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_nameEdit</tabstop>
  <tabstop>m_fileButton</tabstop>
  <tabstop>m_fadeInEdit</tabstop>
  <tabstop>m_fadeOutEdit</tabstop>
  <tabstop>m_audioDevCombo</tabstop>
  <tabstop>m_speedDialButton</tabstop>
  <tabstop>m_previewButton</tabstop>
 </tabstops>
 <resources>
  <include location="qlcui.qrc"/>
  <include location="qlcui.qrc"/>
 </resources>
 <connections/>
</ui>
